include/libtorrent/packet_pool.hpp

```diff
@@ -89,7 +89,7 @@ namespace libtorrent
 		}
 	};
 
-	static packet *create_packet(int size)
+	inline packet *create_packet(int size)
```

Comment:
this broke the documentation generation tool/rst.

https://travis-ci.org/arvidn/libtorrent/jobs/199527166#L1455

My guess is because you put the asterisk to the right instead of to the left (where it really belongs). It seems a little bit far fetched though, but this didn't break before, right?